grabbit/src/test/groovy/com/twcable/grabbit/spring/batch/repository/JcrExecutionContextDaoSpec.groovy

```diff
@@ -102,6 +101,26 @@ class JcrExecutionContextDaoSpec extends Specification {
         result.containsKey("deserialized")
     }
 
+    @Ignore('TODO: Implement this test when Jackalope implements resourceResolver.findResources() API')
```

Comment:
@sagarsane if waiting on this, maybe worth logging an issue: https://github.com/TWCable/jackalope/issues

Don't currently see this problem in the list.